server/core/src/https/v1_scim.rs

```diff
@@ -366,37 +416,18 @@ pub fn route_setup() -> Router<ServerState> {
         //                                                   POST.
         //  -- Kanidm Resources
         //
+        //  Entry    /Entry/{id}      GET                    Retrieve a generic entry
+        //                                                   of any kind from the database.
+        //                                                   {id} is any unique id.
+        .route("/scim/v1/Entry/:id", get(scim_entry_id_get))
```

Comment:
Is the capital-E Entry part of the SCIM spec? Seems weird in our API.

Reply:
Scim spec. But I'm happy to "disregard the authorities" here on this as I hate it also.